src/data/memberships.js

```diff
@@ -91,13 +159,23 @@ export default {
     destinationType: 'meeting',
     members: [
       {
-        personID: 'user1', muted: false, inMeeting: true, orgID: 'org1',
+        personID: 'user1',
+        muted: false,
+        inMeeting: true,
+        sharing: true,
+        orgID: 'org1',
       },
       {
-        personID: 'user2', muted: true, inMeeting: true, orgID: 'org1',
+        personID: 'user2',
+        muted: true,
+        inMeeting: true,
+        sharing: false,
+        orgID: 'org1',
       },
       {
-        personID: 'user3', inMeeting: false, orgID: 'org1',
+        personID: 'user3',
```

Comment:
could you please add `muted: false` for this one so that all members have all fields.

and, only if you're willing, can you put the fields for each member in the same order as the one in the interface:
- personID
- orgID
- muted
- sharing
- inMeeting
https://github.com/webex/component-adapter-interfaces/blob/master/src/MembershipsAdapter.js#L8

thank you!

Reply:
yes, I will added the muted property to false for user3 and also, I will arrange them correctly according to the interface.